tests/test_dtypes.py

```diff
@@ -0,0 +1,90 @@
+import jax.numpy as jnp
+import numpy as np
+import pytest
+import tensorflow as tf
+import torch
+
+from geometric_kernels.kernels.geometric_kernels import MaternKarhunenLoeveKernel
+from geometric_kernels.spaces.circle import Circle
+from geometric_kernels.spaces.hypersphere import Hypersphere
+from geometric_kernels.spaces.mesh import Mesh
+
+
+def to_typed_ndarray(value, dtype):
+    if dtype == "float32":
+        return value.astype(np.float32)
+    elif dtype == "float64":
+        return value.astype(np.float64)
+    else:
+        raise ValueError("Unknown dtype: {}".format(dtype))
+
+
+def to_typed_tensor(value, backend):
+    if backend == "tensorflow":
+        return tf.convert_to_tensor(value)
+    elif backend == "torch":
+        return torch.tensor(value)
+    elif backend == "numpy":
+        return value
+    elif backend == "jax":
+        return jnp.array(value)
+    else:
+        raise ValueError("Unknown backend: {}".format(backend))
+
+
+def mesh_point():
+    n_vertices = 10
+    vertices = np.array(
+        [(1.0 * (i % 2), 1.0 * (i // 2), 0.0) for i in range(n_vertices)]
+    )
+    faces = np.array([(i, i + 1, i + 2) for i in range(n_vertices - 2)])
+    mesh = Mesh(vertices, faces)
```

Comment:
What's the geometric shape of this mesh?

Reply:
Good point, fixed